#! /bin/sh /usr/share/dpatch/dpatch-run
## text_view_typing_error.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid a typing error with recent versions of lablgtk2.

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./ide/undo.ml /tmp/dpep-work.h2rZKK/coq-8.0pl2/ide/undo.ml
--- ./ide/undo.ml	2004-07-16 21:30:21.000000000 +0200
+++ /tmp/dpep-work.h2rZKK/coq-8.0pl2/ide/undo.ml	2005-09-07 21:19:54.000000000 +0200
@@ -18,7 +18,7 @@
   | Insert (s,i,l) -> Delete (s,i,l)
   | Delete (s,i,l) -> Insert (s,i,l)
 
-class undoable_view (tv:Gtk.text_view Gtk.obj) =
+class undoable_view (tv:[> Gtk.text_view ] Gtk.obj) =
   let undo_lock = ref true in 
 object(self)
   inherit GText.view tv as super
diff -urNad --exclude=CVS --exclude=.svn ./ide/undo.mli /tmp/dpep-work.h2rZKK/coq-8.0pl2/ide/undo.mli
--- ./ide/undo.mli	2005-01-21 18:21:33.000000000 +0100
+++ /tmp/dpep-work.h2rZKK/coq-8.0pl2/ide/undo.mli	2005-09-07 21:20:07.000000000 +0200
@@ -10,7 +10,7 @@
 
 (* An undoable view class *)
 
-class undoable_view : Gtk.text_view Gtk.obj ->
+class undoable_view : [> Gtk.text_view ] Gtk.obj ->
 object
   inherit GText.view
   method undo : bool
